helper/helper_test.go

```diff
@@ -504,20 +575,84 @@ var _ = Describe("helper tests", func() {
 			Expect(err).To(HaveOccurred())
 			Expect(err.Error()).To(Equal(msg))
 		})
+		It("CopyData, readerType is SUBSET. Normal completion", func() {
+			writer = bufio.NewWriterSize(&limitWriter{100}, 5)
+
+			test_reader := RestoreReader{
+				readerType: SUBSET,
+				bufReader:  bufio.NewReader(&limitReader{100, io.EOF}),
+			}
+
+			bytesRead, err := test_reader.copyData(18)
+			Expect(bytesRead).To(Equal(int64(18)))
+			Expect(err).ToNot(HaveOccurred())
+		})
+		It("CopyData, readerType is SUBSET. Error on write", func() {
+			*onErrorContinue = true
```

Comment:
Isn't this set before every test?

Reply:
Fixed

Reply:
But then AFTER the test the values ​​of global variables will NOT be reset.